stubs/requests/requests/adapters.pyi

```diff
@@ -29,9 +29,10 @@ SSLError = exceptions.SSLError
 ProxyError = exceptions.ProxyError
 RetryError = exceptions.RetryError
 
-DEFAULT_POOLBLOCK: Any
-DEFAULT_POOLSIZE: Any
-DEFAULT_RETRIES: Any
+DEFAULT_POOLBLOCK: bool
+DEFAULT_POOLSIZE: int
+DEFAULT_RETRIES: int
+DEFAULT_POOL_TIMEOUT: None
```

Comment:
This is defined here: https://github.com/psf/requests/blob/2c2138e811487b13020eb331482fb991fd399d4e/requests/adapters.py#L51-L53

Reply:
It was added in https://github.com/psf/requests/pull/2661 which mentions changing it, and before that, it was same as the local `timeout` variable which was `TimeoutSauce`:

```suggestion
DEFAULT_POOL_TIMEOUT: Optional[TimeoutSauce]
```

The latest requests does  `from urllib3.util import Timeout as TimeoutSauce`, but seems like here in typeshed you need to import from `.packages.urllib3.util`.

Reply:
Fixed! Thanks for pointing that out

Reply:
Actually, after reading the issue I linked and some urllib3 source code, I think this should be `Optional[float]`.

Reply:
I think I figured how this works:
- requests passes this to `_get_conn` in urllib3, defined in `connectionpool.py`
- `_get_conn` passes this to `self.pool.get` in `HTTPConnectionPool`
- `self.pool` has type `self.QueueCls`
-  `ConnectionPool` inherits from `HTTPConnectionPool` which sets `QueueCls = queue.LifoQueue`
- standard-library module `queue` defines the timeout as `Optional[float]`

tl;dr: this should be `Optional[float]`

Reply:
Fixed again. Thanks for the spelunking